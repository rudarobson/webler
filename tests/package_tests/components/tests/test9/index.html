<!--components:
	import .;
-->
<!DOCTYPE html>
<html>

<body>
  <web-grid>
    <web-row>
      <web-col class="s1"></web-col>
    </web-row>
  </web-grid>
</body>

</html>
