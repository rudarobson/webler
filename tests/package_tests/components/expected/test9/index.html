<!DOCTYPE html><html><body><div class="grid"><div class="row"><div class="col s1"></div></div></div></body></html>